context/app/static/js/shared-styles/tables/ExpandableRow/ExpandableRow.jsx

```diff
@@ -1,18 +1,14 @@
-import React, { useRef } from 'react';
-
-import { animated } from 'react-spring';
```

Comment:
Across the whole project it looks like we have 7 uses of `react-spring` -- Would it be worth making a house-keeping issue to follow up and replace those others as well? Or is it still the best tool for some jobs?

Reply:
Still the best tool for some jobs.